packages/sirius-web/backend/sirius-web-starter/src/main/java/org/eclipse/sirius/web/starter/EditingContextDispatcher.java

```diff
@@ -48,6 +63,22 @@ public Mono<IPayload> dispatchQuery(String editingContextId, IInput input) {
 
     @Override
     public Mono<IPayload> dispatchMutation(String editingContextId, IInput input) {
+
+        /*
+         * Since we don't support the capability for libraries modification, capability voters are only checked when the editing context ID is bound to a project.
+         */
+        var optionalProjectId = new UUIDParser().parse(editingContextId)
+                .flatMap(semanticDataId -> this.projectSemanticDataSearchService.findBySemanticDataId(AggregateReference.to(semanticDataId)))
+                .map(ProjectSemanticData::getProject)
+                .map(AggregateReference::getId);
+
+        if (optionalProjectId.isPresent()) {
+            var hasCapability = this.capabilityVoters.stream().allMatch(voter -> voter.vote(SiriusWebCapabilities.PROJECT, optionalProjectId.get(), SiriusWebCapabilities.Project.CREATE) == CapabilityVote.GRANTED);
```

Comment:
I don't see why there should be a link between the ability to change an editing context and the creation of a project, this has nothing to do here. This code should not be modified like that.

Reply:
I will do this change in another PR